src/plugins/application_config/server/index.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { PluginConfigDescriptor, PluginInitializerContext } from '../../../core/server';
+import { ApplicationConfigSchema, configSchema } from '../config';
+import { ApplicationConfigPlugin } from './plugin';
+
+// This exports static code and TypeScript types,
+// as well as, OpenSearch Dashboards Platform `plugin()` initializer.
```

Comment:
nit: can we use block comments here for better readability? `/*....*/`

Reply:
Updated.